packages/maskbook/src/plugins/Wallet/apis/debank.ts

```diff
@@ -99,6 +99,8 @@ export interface HISTORY_RESPONSE {
 }
 
 export async function getTransactionList(address: string) {
-    const response = await fetch(`${DEBANK_API}/history/list?user_addr=${address.toLowerCase()}`)
+    const response = await fetch(`${DEBANK_API}/history/list?user_addr=${address.toLowerCase()}`, {
```

Comment:
i think, use the https://github.com/balazsbotond/urlcat library can prettier code

->
```js
urlcat(DEBANK_API, 'history/list', { user_addr: address.toLowerCase() })
```

Reply:
Better move to next refactor backlog.

Reply:
#1938